extensions/cpp/language-configuration.json

```diff
@@ -7,7 +7,9 @@
 		["{", "}"],
 		["[", "]"],
 		["(", ")"],
-		["#if","#endif"],
+		["#ifndef", "#endif"],
+		["#ifdef", "#endif"],
+		["#if", "#endif"],
```

Comment:
Should the trailing `,` be removed in order to comply with standard json parsing rules? A user was complaining about that at https://github.com/microsoft/vscode/pull/159997#issuecomment-1250634995